pandas/core/frame.py

```diff
@@ -1633,9 +1633,13 @@ def from_records(
         """
         Convert structured or record ndarray to DataFrame.
 
+        Creates a DataFrame object from a structured ndarray, sequence of
+        tuples, sequence of dicts, or DataFrame.
+
         Parameters
         ----------
-        data : ndarray (structured dtype), list of tuples, dict, or DataFrame
+        data : ndarray (structured dtype), sequence of tuples, sequence of dicts, or DataFrame
```

Comment:
Reviewers: this line is over the length limit. I tried breaking the line and indenting the second part of the line in a few different ways, but it caused the docstrings to render incorrectly. I had to add `noqa: E501` below so that this docstring can exceed the line length limit. If you have a better solution, please suggest it!